docs/utils/iers.rst

```diff
@@ -0,0 +1,165 @@
+.. _utils:
```

Comment:
@taldcroft , if you are talking about `.../utils/index.rst:5: WARNING: duplicate label utils, other instance in .../utils/iers.rst` Sphinx warning, it means that this label has been used. Change it to something more unique like `.. _utils-iers:` would fix the warning. Of course, you also have to change any references to this particular label (but not the other conflicting one).